examples/connect/beta-migration/README.md

```diff
@@ -0,0 +1,51 @@
+### Off-Host Execution Beta User Migration
+
+The off-host execution beta moving to GA also moves your content from Bionic to
+Jammy by default. This will cause all content to get rebuilt. If all content being
+rebuilt is acceptable, then no further action is necessary.
+
+If you prefer a phased approach to migration, you may continue using the bionic
+images for existing content and jammy images for new content.
+
+> Note: All images used by existing content should be specified in either
+`launcher.customRuntimeImages` or `launcher.additionalRuntimeImages`. If content
+requires both bionic and jammy images, then you must use R source packages.
+Binary packages will not work when multiple OS distributions are used.
+
+1. If you are currently using `customRuntimeImages: 'base'` (the default),
+apply these changes with:
+
+```
+helm upgrade --install -f https://raw.githubusercontent.com/rstudio/helm/main/examples/connect/beta-migration/values-base.yaml
```

Comment:
Sorry, I used pseudocode on these. Still need a release name and chart reference 🙈 Happy to drop it if you don't think this is worth it. 
```suggestion
helm upgrade --install my-release rstudio/rstudio-connect --devel -f https://raw.githubusercontent.com/rstudio/helm/main/examples/connect/beta-migration/values-base.yaml
```

Reply:
I went ahead and removed this for now. Since these are choices that must be made during a migration I think it's best if users have to be deliberate about choosing images they want to use.